src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Collections/IntervalTreeHelpers.cs

```diff
@@ -171,12 +115,15 @@ public static bool Any<TIntrospector>(TIntervalTree tree, int start, int length,
     {
         var witness = default(TIntervalTreeWitness);
 
+        if (start == int.MinValue && end == int.MaxValue)
```

Comment:
>if (start == int.MinValue && end == int.MaxValue) [](http://example.com/codeflow?start=8&length=49)

was this a bug from before or newly necessary?

Reply:
No bug. But it allows GetEnumerator to work (it wants all nodes and doesn't need/want to pass in an introspector.  (It can't either since the callers don't have one).

Reply:
Note; i doc'ed this on Enumerator side so it can be clear that it is intentionally passing in the full span, but not bothering with any sort of introspector (since it doesn't have one).